nemo_skills/inference/server/model.py

```diff
@@ -177,7 +177,11 @@ def get_generations(self, generation_ids: list[str]) -> list[dict]:
             if not future.done():
                 output = {'generation': None}
             else:
-                output = future.result()
+                try:
```

Comment:
Should we have a parameter that enables this behavior? I'm not entirely sure we should always ignore any errors and keep generating as sometimes the errors are not recoverable and there is no point in generating more if every single example is going to fail with an error. Perhaps by default we raise an error and if some parameter is specified we put it in dict and keep going?